torchbenchmark/util/extra_args.py

```diff
@@ -110,8 +114,11 @@ def apply_decoration_args(model: 'torchbenchmark.util.model.BenchmarkModel', dar
         torch.backends.cuda.matmul.allow_tf32 = True
         torch.backends.cudnn.allow_tf32 = True
     elif dargs.precision == "amp":
+        if model.device == "cpu":
```

Comment:
I think we should make `enable_amp()` to be the default option if it is available, because we should prioritize the model's specific case to support the precision.
```suggestion
        if hasattr(model, "enable_amp"):
           model.enable_amp()
        elif model.device == "cpu":
            import torch
            model.add_context(lambda: torch.cpu.amp.autocast(dtype=torch.bfloat16))
```

Reply:
Done.